packages/db/bin/server.js

```diff
@@ -3,7 +3,9 @@ const { serve } = require("@truffle/db");
 
 const port = 4444;
 
-const server = serve(Config.detect());
+const server = serve(Config.detect({
+  workingDirectory: process.argv[2] || process.cwd()
```

Comment:
this works? doesn't need to be `working_directory`? that'd be swell!

Reply:
Yep! I actually just reverted it back to what it was before I edited it but just confirmed myself that `@truffle/config` handles this. See https://github.com/trufflesuite/truffle/blob/develop/packages/config/src/index.ts#L147-L149